.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
* [ ] Never commit .env file to repo, add the path to your `gitignore` so they don't get commit to the remove branch